src/CtaLink/CtaLink.levers.js

```diff
@@ -0,0 +1,38 @@
+// Levers shared by both CTA Links and CTA Link Button components.
+// ---------------------------------------------------------------------------
+
+/**
+ * Display type
+ */
+export const ctaLinkDisplay = ['inline-block', 'block', 'flex'];
```

Comment:
This might be prop overkill. This can be sent in using className

Reply:
I'll get rid of the display prop